app/lib/pre-apply-handler.js

```diff
@@ -23,7 +27,7 @@ export const preApplyHandler = async (request, h) => {
 
     request.logger.setBindings({ sbi: organisation.sbi });
 
-    if (application?.status === "AGREED" && !application.applicationRedacts.length) {
+    if (application?.status === "AGREED" && !application.redacted) {
```

Comment:
Can we do a qucik sense check on this behaviour here, as I just updated this in the FCP version to track an exception and do a specific redirect, rather than throw here and wait for the error plugin to route the user somewhere. It was needed there as the user ended up in an infinite loop, but it might not be the case here as we're in 1 UI. Could do with verifying this though